src/main/java/org/sasanlabs/service/vulnerability/xss/reflected/XSSInImgTagAttributeInjectionVulnerability.java

```diff
@@ -21,17 +24,21 @@
  * @author KSASAN preetkaran20@gmail.com
  */
 @VulnerableAppRestController(descriptionLabel = "XSS_VULNERABILITY", value = "XSSInImgTagAttribute")
-public class UrlParamBasedImgTagAttrInjection {
+public class XSSInImgTagAttributeInjectionVulnerability {
```

Comment:
Hi @jpralle ,

can you please make the class name same as the value parameter of `@VulnerableAppRestController` annotation? 
Name of the class can be `XSSInImgTagAttribute`